### PR TITLE
Fix Munki tables when munki isn't installed

### PR DESCRIPTION
**Pull request by directionless**
When munki isn't installed, it's plist file obviously doesn't exist. Other osquery tables return no results for non-exist files. Now this does as well.

Along the way, refactor and add some tests.

### CODE REVIEW

File: pkg/osquery/tables/munki/munki_test.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+		})
+	}
+}
+
+func TestGenerateMunkiInstalls(t *testing.T) {
+	t.Parallel()
+
+	//m := New()
+}
```
Comment (blaedj) [inline pkg/osquery/tables/munki/munki_test.go:98]:
```suggestion
```

Comment (directionless) [reply pkg/osquery/tables/munki/munki_test.go]: oops. Good catch

**Comment by blaedj** [comment]
@directionless are you able to repro the original issue locally? The error doesn't seem to pop up for me locally on master, but I may be doing something wrong. I assumed building on master, then running `make osqueryi-tables` and `select * from kolide_munki_installs;` would generate an error, but I don't see one.

**Comment by directionless** [comment]
> @directionless are you able to repro the original issue locally? The error doesn't seem to pop up for me locally on master, but I may be doing something wrong. I assumed building on master, then running `make osqueryi-tables` and `select * from kolide_munki_installs;` would generate an error, but I don't see one.

Yes. 

The code is pretty clear about this error. But I did reproduce locally.

**Comment by blaedj** [comment]
Yep I thought the error source was pretty clear too, but I was a bit surprised I couldn't verify the error.

**Pull request merged**